squire_lib/src/tournament.rs

```diff
@@ -918,6 +918,59 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(self, css: String) -> String {
+        let mut ret: String = "<!DOCTPYE HTML>".to_string();
+
+        ret += "<html lang=\"en\">";
```

Comment:
In general, we should avoid these `+=` for strings as each allocates a whole new string. Instead, we can use the `Write` trait (and its handy macro) on strings to simply extend the existing allocation.
```suggestion
        let _ = write!(ret, "<html lang=\"en\">");
```